pandas/core/frame.py

```diff
@@ -8268,7 +8292,9 @@ def merge(
             validate=validate,
         )
 
-    def round(self, decimals=0, *args, **kwargs) -> DataFrame:
+    def round(
+        self, decimals: Union[int, Dict, Series] = 0, *args, **kwargs
```

Comment:
try to avoid generics without type parameters. `Dict[Label, int]`?

Reply:
yes - added